src/commcare_cloud/ansible/perform_system_checks.yml

```diff
@@ -0,0 +1,12 @@
+- hosts: django_manage
+  become: true
+  become_user: cchq
+  tasks:
+  - shell: 'cd {{ code_home }}; {{ virtualenv_home }}/bin/python manage.py check --deploy'
+    register: deploy_check
+  - shell: 'cd {{ code_home }}; {{ virtualenv_home }}/bin/python manage.py check --deploy -t database'
+    register: database_check
+  - name: "manage.py check --deploy"
+    debug: msg={{ deploy_check.stdout }}
```

Comment:
Nit: is it worth moving this task to before the previous one? That way if `manage.py check --deploy ` runs fine but the `manage.py check --deploy -t database` task fails hard for some reason, we still see the output from the first. It's not the most common thing, but that's my rationalization for feeling like these tasks should switch places. Feel free to ignore.

Reply:
Thanks for pointing this out. Turns out these debug tasks will ever get called if either of the shell tasks fails. This needed a lot more debugging and major rework. 69fef4efbab2a27d68bdcd85a8fc326368ee8b9